reed-solomon-novelpoly/src/field/inc_encode.rs

```diff
@@ -190,7 +190,9 @@ pub fn encode_sub_plain(bytes: &[u8], n: usize, k: usize) -> Result<Vec<Additive
 	// so we get a buffer of size `N` in `GF` symbols
 	let mut elm_data = vec![Additive(0); n];
 
-	for i in 0..(bytes_len / 2) {
+	let tmp = if bytes_len % 2 == 0 { bytes_len / 2 } else { (bytes_len + 1) / 2 };
```

Comment:
can it be simplified to
```suggestion
	let tmp = (bytes_len + 1) / 2;
```